site/tests/integration/block-publish-npm.test.ts

```diff
@@ -80,6 +82,8 @@ test("user should not be able to publish an invalid npm-package", async ({
 test("user should not be able to publish an already-used npm-package", async ({
   page,
 }) => {
+  test.skip();
+  /** @todo skip this test if `isProduction === false` */
```

Comment:
`isProduction` will never be false in these tests – we need some other approach, so I suggest updating the todo. We don't have to figure it out now though.
```suggestion
  /** @todo update 'already-taken' name check so that it can be checked in these tests (e.g. by having the check activated by custom environment variable, not `isProduction`) */
```

Reply:
I suggest removing the todo, let me update it